upf/upf_proxy_input.c

```diff
@@ -133,26 +134,48 @@ splice_tcp_connection (flow_entry_t * flow, flow_direction_t direction)
   session_t *s;
 
   if (rev->conn_index == ~0)
-    return UPF_PROXY_INPUT_NEXT_TCP_INPUT;
+    {
+      vlib_increment_simple_counter (&gtm->upf_simple_counters
+				     [UPF_FLOW_NOT_STITCHED_NO_REV_CONNECTION],
+				     vlib_get_thread_index (), 0, 1);
```

Comment:
It is incorrect to increase a counter here b/c the state of the flow may change yet, and rev conn may appear. Meanwhile, the counter may be increased multiple times for the same flow. We only should increase the counters when the stitching is either done or completely and finally rejected for the flow.